shell/platform/common/text_editing_delta.h

```diff
@@ -0,0 +1,72 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_COMMON_TEXT_EDITING_DELTA_H_
+#define FLUTTER_SHELL_PLATFORM_COMMON_TEXT_EDITING_DELTA_H_
+#define _SILENCE_CXX17_CODECVT_HEADER_DEPRECATION_WARNING
+
+#include <codecvt>
+#include <string>
+
+#include "flutter/shell/platform/common/text_range.h"
+
+namespace flutter {
+
+struct TextEditingDelta {
+ public:
+  TextEditingDelta(std::u16string textBeforeChange,
+                   TextRange range,
+                   std::u16string text);
```

Comment:
It looks like `TextEditingDelta`s are copyable types. In this case, you should define:
* A copy constructor: `TextEditingDelta(const TextEditingDelta& other)`
* operator=: `TextEditingDelta& operator=(const TextEditingDelta& other)`

In this case, you should be able to declare them as `default`.

Ref: https://google.github.io/styleguide/cppguide.html#Copyable_Movable_Types